examples/with-vite/packages/ui/utils/counter.ts

```diff
@@ -0,0 +1,9 @@
+export function setupCounter(element: HTMLButtonElement) {
+  let counter = 0;
+  const setCounter = (count: number) => {
+    counter = count;
+    element.innerHTML = `count is ${counter}`;
```

Comment:
Just for safety even though it doesn't actually make a huge difference:

```suggestion
    element.innerText = `count is ${counter}`;
```

Reply:
I like it